temp_delete_me.md

```diff
@@ -0,0 +1,2 @@
+::: STUFF TO DO :::
+- Load test the worker with a petitioner/practitioner that is associated with many cases on the ustc/test environment.
```

Comment:
will be doing this in `test` :)

Reply:
awesome, we can probably remove this, will confirm with Zach, I know he was looking forward to the removal of this file!